pubsub/google/cloud/pubsub_v1/subscriber/policy/thread.py

```diff
@@ -263,7 +263,7 @@ def open(self, callback):
         # Create the Future that this method will return.
         # This future is the main thread's interface to handle exceptions,
         # block on the subscription, etc.
-        self._future = Future(policy=self)
+        self._future = Future(policy=self, event_factory=threading.Event)
```

Comment:
@lukesneeringer I was being **intentionally** explicit here, i.e. I want the `threading` implementation to be clear it is using a `threading.Event`